third_party/blink/renderer/core/html/media/html_media_element_test.cc

```diff
@@ -1568,4 +1574,41 @@ TEST_P(HTMLMediaElementTest, CanFreezeWithMediaPlayerAttached) {
   EXPECT_FALSE(MediaIsPlaying());
 }
 
+#if BUILDFLAG(USE_STARBOARD_MEDIA)
+TEST(HTMLMediaElementTest, CanHandleCobaltProgressiveSupportQueries) {
```

Comment:
Are blink unit tests fully supported on Android? This test crashes on Android but not in Linux

Reply:
Don't want to spend too much time investigating if I'm not sure if the tests have been validated yet.

Reply:
I've removed the test to unblock this. The test is [saved](https://paste.googleplex.com/6093793116553216), and once we validate blink tests on Android I'll re-add it.

Reply:
Re added since this is passing on Linux